examples/UEFA/UEFA.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <!DOCTYPE token  [
-        <!ENTITY UEFA.en SYSTEM "UEFA.en.shtml">
-        <!ENTITY UEFA.zh SYSTEM "UEFA.zh.shtml">
+        <!ENTITY UEFA.en SYSTEM "UEFA.en.js">
+        <!ENTITY UEFA.zh SYSTEM "UEFA.zh.js">
         <!ENTITY style SYSTEM "shared.css">
         ]>
 <ts:token xmlns:ts="http://tokenscript.org/2020/03/tokenscript"
```

Comment:
@colourful-land all of these are 404's, did you intend to put the files in there?

Reply:
> @colourful-land all of these are 404's, did you intend to put the files in there?

No need to put anything there since it's a namespace, not link. I will still put something there one day to help ease the learning curve but not in this PR.